pkgs/games/curseofwar/default.nix

```diff
@@ -1,28 +1,20 @@
 { stdenv
 , fetchFromGitHub
-, fetchpatch
 , ncurses
 , SDL
 }:
 
 stdenv.mkDerivation rec {
   pname = "curseofwar";
-  version = "1.2.0";
+  version = "1.3.0";
 
   src = fetchFromGitHub {
     owner = "a-nikolaev";
     repo = pname;
     rev = "v${version}";
-    sha256 = "1bj3lv5vrnwzzkgj31pyf1lzkz10qphvzlfz7a3j4plqkczjq92y";
+    sha256 = "1wd71wdnj9izg5d95m81yx3684g4zdi7fsy0j5wwnbd9j34ilz1i";
   };
```

Comment:
This could fix the macOS build: https://github.com/a-nikolaev/curseofwar/blob/3386ae847e5d674fa79136c567f27eef962e0b9b/Makefile#L2

```
  makeFlags = [
    "CC=${stdenv.cc.targetPrefix}cc"
  ];
```

Reply:
Thank you!